content/blog/jjpx/2018-09-05_haiku_monthly_activity_report_august_2018.md

```diff
@@ -0,0 +1,43 @@
++++
+type = "blog"
+title = "Haiku monthly activity report - 08/2018"
+author = "jjpx"
+date = "2018-09-05 16:37:40-03:00"
+tags = []
++++
+
+Time has come and a new monthly activity report arrived with it. It is the first time I write something this long in english. Enjoy!
+
+This report covers hrev52141-52310.
+
+### Drivers
+
+SDHCI MMC driver from [krish_iyer](https://www.haiku-os.org/blog/krish_iyer) has been merged in master. There is still a lot of work to do and it is not enabled in automated builds yet, but should serve as a basis for any future related work. His commits include extensive documentation on how to setup a testing vm, what has been done and what is left to do.
+
+There have been several contributions to the ARM port including work by kallisti5 on our implementation fo Flattened Device Tree (FDT) that should make it easier for Haiku to be run on more ARM hardware variants and a rewrite of the memcpy function for ARM devices by Augustin Cavalier based on Clang 8.
+
+Finally on the driver related news kallisti5 fixed an SMAP violation on s3 accelerant.
```

Comment:
You need to look at the "author" for commits, not the "commiter" (which is the person who reviewed the change). S3 fixes were made by korli (Jérôme Duval).

Reply:
It's not "an SMAP violation", sorry for the wrong commit message. It's a follow up on commit https://git.haiku-os.org/haiku/commit/?id=023a547dc6981f10bb8437cfba7fcaa3fe3b0d01 and  lets the accelerant clone in userland the areas created by the kernel driver.